server/handlers/image_handler.py

```diff
@@ -1,19 +1,53 @@
+import logging
+import socket
 from random import random, randint, choice
 import string
 
 
 class ImageHandler:
-    def __init__(self, gs):
+    def __init__(self, gs, config):
+        self.logger = logging.getLogger("main")
         self.gs = gs
+        self.config = config
+        print("╠ CREATED IMAGE HANDLER")
+        self.logger.info("╠ CREATED IMAGE HANDLER")
```

Comment:
I noticed these and thought they were weird. Perhaps you should have some sort of namespacing thing or something that indicates the module. e.g. `"[Image] CREATED IMAGE HANDLER"` or `"groundstation.image CREATED IMAGE HANDLER"` (I think that's more Java-like namespacing.) In fact, the former example is used later in this same file.

Reply:
The reason for this is simply to make a cool pattern for initialization: if you try running it, you'll see what I mean. It's solely for stylish purposes.

I felt like this wasn't really a log from a process in the class, just when it was started and initialized. It fits in with the pattern super well - I can't really describe it that well, so you should try running it if you want to see.